src/mono/wasm/runtime/assets.ts

```diff
@@ -76,72 +126,65 @@ export async function mono_download_assets(): Promise<void> {
     }
 }
 
-export async function start_asset_download(asset: AssetEntry): Promise<AssetEntry | undefined> {
+export async function start_asset_download(asset: AssetEntryInternal, downloadData: boolean): Promise<ArrayBuffer | undefined> {
     try {
-        return await start_asset_download_throttle(asset);
+        return await start_asset_download_throttle(asset, downloadData);
```

Comment:
This whole function should be refactored to support `n` retries. That should work for n==0 too.

Reply:
different PR